chronos/ChronosAutoImport/runImport.bat

```diff
@@ -0,0 +1,3 @@
+cd C:\Users\flyzoor\Documents\Projects\ChronosAutoImport
```

Comment:
```suggestion
cd C:\Theseus\Tools\ChronosAutoImport
```

Reply:
Agreed, this should be changed when project goes live. Currently it is like this for testing purposes.